mobile/login/login.tsx

```diff
@@ -77,10 +77,13 @@ const Login = ({ navigation }) => {
                 <Text style={login_styles.imdown_title}>imdown</Text>
                 <Text style={login_styles.imdown_description}>A better way to manage group events.</Text>
             </View>
-            <View style={login_styles.google_sign_in_button}>
-                <TouchableOpacity onPress={onPress}>
-                    <Image source={require('../assets/img/sign_in_with_google.png')} />
-                </TouchableOpacity>
+            <View style={login_styles.google_sign_in_button_container}>
+                <GoogleSigninButton
+                    style={login_styles.google_sign_in_button}
+                    size={GoogleSigninButton.Size.Wide}
+                    color={GoogleSigninButton.Color.Light}
```

Comment:
(Commenting because I had not paid attention to the task board, and had already done some work on this page while implementing android sign in in a pending PR)

I think that you don't need to set the color to light if you don't want to, because it's the default value.
Additionally, is there a reason you are adding a new style that overrides the suggested sizes (just wondering - if you did this on purpose that's cool but I noticed that you just copied the demo style values from the instruction page)? My personal opinion is that Wide looks best when there is 1 sign-in option and Standard looks best when there are multiple due to the difference in the description string.

Reply:
Also you should delete those old image files